### ISSUE TITLE
textarea resize not taking line-height into consideration

### ISSUE THREAD

**Issue by icellan** [Dogfalo/materialize#3135]
When a textarea has a line-height defined for it, the resizing of the textarea does not work properly.

Adding the line-height css attribute fixes this, just below "hiddenDiv.css('box-sizing'....":
```
            hiddenDiv.css('line-height', $textarea.css('line-height'));
```

**Comment by acburst** [comment]
Fixed in ad28884

**Comment by temuujina** [comment]
Is the ad28884 change in materialize  v0.97.6 which is distributed presently ?
